me.brokenby.PdfMetadataEditor.yml

```diff
@@ -0,0 +1,38 @@
+app-id: me.brokenby.PdfMetadataEditor
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+commands:
+  - '/app/bin/pmedit-cli'
+modules:
+  - name: pdf-metadata-editor-app
+    buildsystem: simple
+    build-commands:
+      - mv -v * ${FLATPAK_DEST}
+    sources:
+      - type: archive
+        url: https://github.com/zaro/pdf-metadata-editor/releases/download/v3.0.0/pdf-metadata-editor-3.0.0-linux-portable.tar.gz
+        sha256: 80303e459c1c3b6ecf13923545ca349a2c3e1ff2a3ede08644adba98b2a9de2d
+  - name: pdf-metadata-editor-desktop
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/share/icons/hicolor/128x128/apps
+      - mkdir -p /app/share/mime/packages/
+      - mkdir -p /app/share/applications/
+      - cp -v pdf-metadata-edit.png /app/share/icons/hicolor/128x128/apps/me.brokenby.PdfMetadataEditor.png
+      - cp -v me.brokenby.PdfMetadataEditor-mime.xml /app/share/mime/packages/me.brokenby.PdfMetadataEditor.xml
+      - cp -v *.desktop /app/share/applications/
+    sources:
+      - type: dir
+        path: flatpak
+
+finish-args:
+  [
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=pulseaudio",
+    "--share=network",
+    "--device=dri",
+    "--filesystem=home",
```

Comment:
Why do you need full home? Does it support filechooster portals?

Reply:
I am really not sure how to answer . If I remove the --filesystem=home, the home directory  is empty, and the whole purpose of the tool is to edit existing files. 

Not sure also about the filechooser portals, but unless Java 17 supports them the answer is probably no.